htdocs/product/price.php

```diff
@@ -1452,7 +1452,7 @@ function on_change() {
         $num = $db->num_rows($result);
 
     	if (!$num)
-    	{
+    	{/*
```

Comment:
The goal of this code is to be sure there is always at least 1 line into product_price table.
If not we must fix this, then this code will do nothing anymore. If you remove this, you remove the "automatic" fix that fixes corrupted data (price into table llx_product must be same than the line into product_price for level = 1). 


Do you use PRODUIT_MULTIPRICES (option to have several level prices per segment of customers) ? 
- If yes, you must import your prices using the import profile "Customer prices (of products or services, multi-prices)"
- If no, this should have no effect, except copying the price into llx_prouct table as a new line into llx_product_price

Reply:
Hi Eldy,
I don't use PRODUIT_MULTIPRICES option.
Your fix [73dfa7d](https://github.com/Dolibarr/dolibarr/commit/73dfa7d44a115b153b4b421e23ba6ae5da2c769d) solves perfectly my problem and the history is now identical to the price imported so this historical line does not break my imported selling price.
Thanks a lot.
In fact my patch was just a workaround to stop the trouble for my users.
I thought there was something wrong in the the code `$object->updatePrice($object->price,` because my imported prices was TTC...
and I saw that you upgraded it with TTC notions, so I was sure it could be better ;-)
Have a good day.